Sources/SnapshotTesting/Cocoa.swift

```diff
@@ -12,33 +12,50 @@
     }
 
     public var diffableData: Data {
+      let scale = NSScreen.main!.backingScaleFactor
+      self.size = .init(width: self.size.width * 2.0 / scale, height: self.size.height * 2.0 / scale)
       let cgImage = self.cgImage(forProposedRect: nil, context: nil, hints: nil)!
       let rep = NSBitmapImageRep(cgImage: cgImage)
       rep.size = self.size
       let data = rep.representation(using: .png, properties: [:])!
       return data
     }
 
-    public func diff(comparing other: Data) -> XCTAttachment? {
-      let existing = NSImage(data: other)!
+    public func diff(from other: NSImage) -> Bool {
+      let bitmapInfo = NSImage(data: self.diffableData)!
```

Comment:
Have to convert the image to data and back again for the `bitmapInfo`s to match :(

Up for suggestions on optimizing here.